mkstage4.sh

```diff
@@ -10,10 +10,20 @@ fi
 #set flag variables to null
 EXCLUDE_BOOT=0
 EXCLUDE_CONNMAN=0
+EXCLUDE_LOST=1
 QUIET=0
+USAGE="usage:\n\
+  `basename $0` [-q -c -b] [-s || -t <target-mountpoint>] <archive-filename> [custom-tar-options]\n\
+  -q: activates quiet mode (no confirmation).\n\
+  -c: excludes connman network lists.\n\
+  -b: excludes boot directory.\n\
+  -l: includes lost+found directory.\n\
```

Comment:
I do, but I'd rather give up on that than on argument consistency. After all, people will probably use this only infrequently, so we have to keep it as simple and comprehensible as possible.